# This contains output of an optimizer including optimized footholds and
# optmized spline coefficients.
# Note that currently only x-y-position of footholds are optimized.

geometry_msgs/Point[] footholds
float64[] spline_coeff
